addons/mail/static/src/core/common/core.scss

```diff
@@ -19,7 +19,7 @@ $o-discuss-talkingColor: lighten($success, 5%);
     background-color: var(--o-discuss-badge-bg) !important;
     align-items: center;
     justify-content: center;
-    min-width: 2.1ch;
+    min-width: 2.9ch;
```

Comment:
This affects all showing of badge, including messaging menu and chat bubble / windows and so on. I haven't tested this change but I have a feeling this puts some other issues like single digit counters are sometimes too large

Reply:
I've tried them and it seems that in all those cases the `width` is greater than the `min-width` (with the new value), so I think this change should not affect them.